### PR TITLE
WiFi Channel Parameter

### PR DESCRIPTION
**Pull request by jlpoltrack**
Add a define for WiFi channel, set to 13 which should overlap least with mLRS freq list and home WiFi.

Tested on my ESP32.

### CODE REVIEW

**Comment by olliw42** [comment]
very good idea
I actually had the very same idea and did some play with it this morning, just didn't added it

would you mind to not do it as a #define but as int wifi_channel = 13; ?

**Comment by jlpoltrack** [comment]
Okay to use 'uint8_t' data type?

**Comment by olliw42** [comment]
good question. Pl use int.
Somehow Arduino seems to uses this more often, and the function softAP() uses int.

**Comment by jlpoltrack** [comment]
Okay :)

File: esp/mlrs-wifi-bridge/mlrs-wifi-bridge.ino [hunk 1]
```diff
@@ -71,6 +71,10 @@ int baudrate = 57600;
 // comment out for default setting
 #define WIFI_POWER  WIFI_POWER_2dBm // WIFI_POWER_MINUS_1dBm is the lowest possible, WIFI_POWER_19_5dBm is the max
 
+// WiFi channel
+// comment out for default setting
+int WIFI_CHANNEL = 13; // Use 13 (2461-2483 MHz) since it has least overlap with 2.4 freqs
```
Comment (olliw42) [inline esp/mlrs-wifi-bridge/mlrs-wifi-bridge.ino:76]: lower case lower case :D

Comment (jlpoltrack) [reply esp/mlrs-wifi-bridge/mlrs-wifi-bridge.ino]: Thanks for the patience :) I see the pattern now:
- lower for variables
- upper for defines

Comment (olliw42) [reply esp/mlrs-wifi-bridge/mlrs-wifi-bridge.ino]: 👍

**Comment by olliw42** [comment]
MANY THX sir !!!

**Pull request merged**